### DISCUSSION TITLE
The type or namespace name 'Sentry' could not be found (are you missing a using directive or an assembly reference?)

### DISCUSSION THREAD

**Discussion by Dilosen** [WalletConnect/WalletConnectUnity#214] [Q&A] [answered]
Missing this using reference - what is the package dependancy that is required to be installed? I tried asking GPT but not 100% if it's the right "sentry" package from Git.

Please could you provide this in the Samples package, out of the box it throws this error: Assets\Samples\WalletConnectUnity Modal\1.1.3\Modal Sample\Scripts\Dapp.cs(4,7): error CS0246: The type or namespace name 'Sentry' could not be found (are you missing a using directive or an assembly reference?)

or alternatively, please could you provide the git url for the required sentry package

**Comment by Dilosen** [comment, marked as answer]
I found this: https://docs.sentry.io/platforms/unity/

I assume this is the package. 

Installed, but now throws another missing reference for User. 

Assets\Samples\WalletConnectUnity Modal\1.1.3\Modal Sample\Scripts\Dapp.cs(49,38): error CS0246: The type or namespace name 'User' could not be found (are you missing a using directive or an assembly reference?)

I see in the Sentry sample it is  

```SentrySdk.ConfigureScope(s =>
        {
            s.User = new SentryUser
            {
                Email = "ant@farm.bug",
                Username = "ant",
                Id = "ant-id"
}

```


changing the sample script from User to SentryUser resolves this issue.

**Reply by skibitsky** [reply]
Yes, sorry about that, you can remove this code. I'll fix the compile issue in the next update.

**Reply by Dilosen** [reply]
no problem, I've never used Sentry before, but I suppose it handles the logging - simple fix by just updating the function being called.

In fact, I have enabled the logging because I am having issues, and I assume that it is Sentry that is logging that information?

**Reply by skibitsky** [reply]
We use Sentry during internal QA. It logs all exceptions that occur during testing. The sample provided with the Modal package doesn't send us any data when you run it.
